### DISCUSSION TITLE
osu crashes and blackscreen

### DISCUSSION THREAD

**Discussion by CH4DBULL** [ppy/osu#16456] [Q&A]
![Screenshot_20220114-160936](https://user-images.githubusercontent.com/53918873/149571873-d1f51eaa-923f-48cc-ad4b-aa39e257e3e0.png)

and get blackscreen when start apk

my phone is an motorola g6 (normal version)

with cpu 32 bits

and 3 gb of ram

with android 9 pie

**Comment by bdach** [comment]
[Please provide logs to help us troubleshoot your problems](https://github.com/ppy/osu/discussions/15188)

**Reply by CH4DBULL** [reply]
> [Please provide logs to help us troubleshoot your problems](https://github.com/ppy/osu/discussions/15188)


![Screenshot_20220114-164052](https://user-images.githubusercontent.com/53918873/149575399-58d0422d-9c59-4b0b-8405-68f4ba440d65.png)
![Screenshot_20220114-164045](https://user-images.githubusercontent.com/53918873/149575409-620ebe02-4bc5-4eec-9e0f-8a3442a17906.png)


I can't find help please

**Reply by bdach** [reply]
what's in the `logs` directory on the second screenshot

**Reply by CH4DBULL** [reply]
> what's in the `logs` directory on the second screenshot

![Screenshot_20220114-165123](https://user-images.githubusercontent.com/53918873/149576562-ebbd0353-7d56-4081-97f5-12165ffeb3b9.png)


Which of these files do I have to pass?

**Reply by bdach** [reply]
all of them would be best

**Reply by CH4DBULL** [reply]
> all of them would be best

ok I'm going to upload them to mediafire I don't know how to upload them to github on mobile

**Reply by CH4DBULL** [reply]
> all of them would be best

https://www.mediafire.com/file/nx6vcv2s9ojduop/database.log/file

https://www.mediafire.com/file/ztgurztj8rp1qhj/performance.log/file



https://www.mediafire.com/file/64rooy5ysr50nsa/runtime.log/file


https://www.mediafire.com/file/zekay85czlahnvr/network.log/file

**Reply by bdach** [reply]
this log looks as if the game was working completely fine. there are beatmap changes listed in it and everything. can you explain that? does the game crash instantly, or only after performing some specific action?

i do not know how to help you any more unless you are able to provide a video of this or logcat output from your android device (do that second part only if you are comfortable with developer stuff).

**Reply by CH4DBULL** [reply]
I'm going to record a video

**Reply by CH4DBULL** [reply]
I am uploading a video regarding performance and black screen

**Reply by CH4DBULL** [reply]
> this log looks as if the game was working completely fine. there are beatmap changes listed in it and everything. can you explain that? does the game crash instantly, or only after performing some specific action?
> 
> i do not know how to help you any more unless you are able to provide a video of this or logcat output from your android device (do that second part only if you are comfortable with developer stuff).

https://www.mediafire.com/file/iviqps5rmvhkxlo/2022_01_14_22_20_03.mp4/file

**Reply by frenzibyte** [reply]
Does turning off beatmap video (i.e. "Storyboard / Video" option in top-right of player loading screen) resolve the issue for you? and/or disabling "hardware video acceleration" in game settings.

**Reply by CH4DBULL** [reply]
> 

It didn't slow me down yet I deactivated it and I got more fps but it felt like 8 I don't know if I'm explaining myself well in the video you can see how the performance is until the application crashes

**Reply by frenzibyte** [reply]
Not sure how that answers my question?

**Reply by CH4DBULL** [reply]
> Not sure how that answers my question?

he answers it but let me summarize the answer so that it is understandable the hardware acceleration option made me slow and let's say it was not optimal and it is not profitable to have it active taking advantage of the multicore, I activated it I got a lot of fps but the application crashed and I got a black screen

**Reply by frenzibyte** [reply]
So does that mean turning "hardware video acceleration" off resolves the issue?

**Reply by CH4DBULL** [reply]
> So does that mean turning "hardware video acceleration" off resolves the issue?

No, but it solves the lag problem for me but now it crashes when I start playing.... which is what you see in the video

**Reply by TheFantasticLoki** [reply]
@yhx7 do you think you could download an app called droid info and when you open it you'll see a file icon up in the top right of the app. Click that and then click text, then select email and copy all the information it pastes into the email and send it here. This may give some more information as to why (from what I can gather) multicore is causing your game to crash with hardware acceleration off.

**Reply by bdach** [reply]
@TheFantasticLoki i fail to see how that would be helpful. also please refrain from asking users to install applications of dubious origin on their phones.

**Reply by TheFantasticLoki** [reply]
Droid info is an app that gives exact information about the hardware inside your device, very useful in certain situations and based off what little I could understand from what he was saying it seemed like he was having crash problems with hardware acceleration off but multicore turned on. So from what I can gather seems like osu maybe trying to use more cores than are available or something to that effect but I figured a detailed report of the exact hardware inside his device would help with diagnosing the problem. Only trying to help
